### ISSUE TITLE
Notify when an updated version of GitBucket has been released

### ISSUE THREAD

**Issue by Fmstrat** [takezoe/gitbucket#687]
Insert a checkbox in the server settings screen to enable a notification of new versions

On administrator login, do a page request on

     https://github.com/takezoe/gitbucket/releases/latest

If the URL does not redirect to 

     https://github.com/takezoe/gitbucket/releases/[current version number]

then the user is not running the latest release.

Lastly, create a dismissable DIV popup in the corner indicating that a new version is available

**Comment by aadrian** [comment]
"Check for update" IMHO should also check the install plug-ins for updates (the same does Eclipse and IntelliJ and other software with update check functionality too).

**Comment by McFoggy** [comment]
@aadrian I fully agree that we should also check the plugins and not just for latest version but also for version compatibility. 

The tricky part here is to detect & ensure upward compatibility. 
Without a 3rd party in the game (I mean here an external registry) or without re-release of plugins (plugins metadata could be enough) it might be difficult to ensure.

**Comment by aadrian** [comment]
This could be done from a plug-in, since it only needs to check for available "newer" resources, not to update them.
